README.md

```diff
@@ -1,4 +1,14 @@
+
+
 # Fedora
+---
+<div style="color:red">
+**PLEASE BE ADVISED:**  this branch (master) is currently the main branch for 6.0 development. It is a  work in progress!  
+While it will build and any unignored tests willpass, the codebase in not currently in working order and will not 
```

Comment:
Typo: "..the codebase [in] not currently..." should be "..the codebase [is] not currently..."